src/matmul.jl

```diff
@@ -7,6 +19,11 @@ function Base.LinAlg.A_mul_B!{T<:BlasFloat}(α::T, A::SparseMatrixCSC{T,BlasInt}
         cscmv!('N',α,matdescra(A),A,B,β,C) :
         cscmm!('N',α,matdescra(A),A,B,β,C)
 end
+function Base.LinAlg.A_mul_B!{T<:BlasFloat}(A::SparseMatrixCSC{T,BlasInt}, B::StridedVecOrMat{T})
+    isa(B,AbstractVector) ?
+        Base.LinAlg.A_mul_B!(one(T),A,B,zero(T),zeros(T, size(A,1))) :
+        Base.LinAlg.A_mul_B!(one(T),A,B,zero(T),zeros(T, size(A,1), size(B,2)))
+end
```

Comment:
Seems like it would be better to have two different methods, one with `B::AbstractVector` and another with `B::AbstractMatrix`. 

Just a style note, maybe import Base.LinAlg.A_mul_B methods and overload them?